packages/synapse-react-client/src/utils/functions/StringUtils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { hex2ascii, normalizeNumericId } from './StringUtils'
+import { JoinTeamSignedToken } from '@sage-bionetworks/synapse-types'
+
+describe('StringUtils', () => {
```

Comment:
This test file was renamed to match the case of the utility file. Only added tests for `normalizeNumericId` below